app/i18n/cz/conf.php

```diff
@@ -13,107 +13,107 @@
 		'help' => 'Více možností je dostupných v nastavení jednotlivých kanálů',
-		'keep_favourites' => 'Never delete favourites',	// TODO
-		'keep_labels' => 'Never delete labels',	// TODO
-		'keep_max' => 'Maximum number of articles to keep',	// TODO
-		'keep_min_by_feed' => 'Zachovat tento minimální počet článků v každém kanálu',
-		'keep_period' => 'Maximum age of articles to keep',	// TODO
-		'keep_unreads' => 'Never delete unread articles',	// TODO
-		'maintenance' => 'Maintenance',	// TODO
+		'keep_favourites' => 'Nikdy neodstraňovat oblíbené',
+		'keep_labels' => 'Nikdy neodstraňovat popisky',	// TODO
+		'keep_max' => 'Maximální počet článků k ponechání',
+		'keep_min_by_feed' => 'Minimální počet článků k ponechání na kanál',
+		'keep_period' => 'Maximální stáří článků k ponechání',
+		'keep_unreads' => 'Nikdy neodstraňovat nepřečtené články',
+		'maintenance' => 'Údržba',
 		'optimize' => 'Optimalizovat databázi',
-		'optimize_help' => 'Občasná údržba zmenší velikost databáze',
-		'policy' => 'Purge policy',	// TODO
-		'policy_warning' => 'If no purge policy is selected, every article will be kept.',	// TODO
-		'purge_now' => 'Vyčistit nyní',
+		'optimize_help' => 'Občas spusťte pro zmenšení velikosti databáze',
+		'policy' => 'Zásady vymazání',
+		'policy_warning' => 'Pokud není vybrána žádná zásada vymazání, budou ponechány všechny články.',
+		'purge_now' => 'Vymazat nyní',
 		'title' => 'Archivace',
-		'ttl' => 'Neaktualizovat častěji než',
+		'ttl' => 'Neobnovovat automaticky častěji než',
 	),
 	'display' => array(
 		'_' => 'Zobrazení',
 		'icon' => array(
 			'bottom_line' => 'Spodní řádek',
-			'display_authors' => 'Authors',	// TODO
+			'display_authors' => 'Autoři',
 			'entry' => 'Ikony článků',
 			'publication_date' => 'Datum vydání',
-			'related_tags' => 'Související tagy',
+			'related_tags' => 'Štítky článků',
 			'sharing' => 'Sdílení',
-			'summary' => 'Summary',	// TODO
+			'summary' => 'Souhrn',
 			'top_line' => 'Horní řádek',
 		),
 		'language' => 'Jazyk',
 		'notif_html5' => array(
-			'seconds' => 'sekund (0 znamená žádný timeout)',
-			'timeout' => 'Timeout HTML5 notifikací',
+			'seconds' => 'sekund (0 znamená žádný časový limit)',
+			'timeout' => 'Časový limit HTML5 oznámení',
 		),
-		'show_nav_buttons' => 'Show the navigation buttons',	// TODO
-		'theme' => 'Vzhled',
-		'theme_not_available' => 'The “%s” theme is not available anymore. Please choose another theme.',	// TODO
+		'show_nav_buttons' => 'Zobrazit navigační tlačítka',
+		'theme' => 'Motiv',
+		'theme_not_available' => 'Motiv „%s“ již není dostupný. Zvolte jiný motiv, prosím.',
 		'thumbnail' => array(
-			'label' => 'Thumbnail',	// TODO
-			'landscape' => 'Landscape',	// TODO
-			'none' => 'None',	// TODO
-			'portrait' => 'Portrait',	// TODO
-			'square' => 'Square',	// TODO
+			'label' => 'Náhled',	// TODO
+			'landscape' => 'Na šířku',	// TODO
+			'none' => 'Žádný',	// TODO
+			'portrait' => 'Na výšku',	// TODO
+			'square' => 'Čtverec',	// TODO
```

Comment:
Could you remove the TODO comment?

Reply:
Sorry, all TODO comments are removed now.